src/signer.ts

```diff
@@ -725,6 +758,10 @@ export class Signer {
         return {handled: false};
       }
 
+      if (isNullish(this.#owner)) {
```

Comment:
This does not sound correct, should an error be throw somewhere if the owner is null at this point?